docs/source/1.0/spec/core/model.rst

```diff
@@ -445,7 +445,9 @@ Shape IDs are formally defined by the following ABNF:
     root_shape_id          :`absolute_root_shape_id` / `identifier`
     absolute_root_shape_id :`namespace` "#" `identifier`
     namespace              :`identifier` *("." `identifier`)
-    identifier             :(ALPHA / "_") *(ALPHA / DIGIT / "_")
+    identifier             :identifier_start *identifier_chars
```

Comment:
Went looking to see if this needs to be updated in the `idl.rst` file of the spec, but it looks like it isn't present there. Should probably add this section of [the ABNF over there as well, since its heading seems to indicate completeness](https://github.com/awslabs/smithy/blob/master/docs/source/1.0/spec/core/idl.rst#smithy-idl-abnf).

Reply:
Sphinx seems broken and doesn't allow duplicate productionlists with different names to coexist. So instead, I just added a link from the IDL to the shape ID section of the model. I think this is more appropriate than the other way around because shape IDs exist in the semantic model, the IDL, and AST.